Feature email notification

diedpigs opened the pull request:
Add email notification form option for vnc desktop
Enable actual email notification support by adding two option passing to scheduler:
 `--mail-type=BEGIN` and `--mail-user=user_email`

jprorama commented:
We should add the @domain string as a variable from groups var all. This may require these files to be templates.  Please comment on viability of this approach. 

If it must be hard coded then targeting a prod repo might be better.

More generally, it may be better to consider introducing a variable USER_EMAIL that could be set by the target system according to site conventions and then just consuming that variable, if set, in this patch.

diedpigs commented:
> We should add the @Domain string as a variable from groups var all. This may require these files to be templates. Please comment on viability of this approach.
> 
> If it must be hard coded then targeting a prod repo might be better.
> 
> More generally, it may be better to consider introducing a variable USER_EMAIL that could be set by the target system according to site conventions and then just consuming that variable, if set, in this patch.

Added `USER_EMAIL` environment variable in code base, need further configuration on system.
Added `mail_domain` in `group_vars/all`
Changed ansible copy module to template module in order to use `mail_domain` variable.

diedpigs commented:
Do you want me to rebase the branch? Or just merge against `feat-openstack`?

rtripath89 commented:
Rebase to feat-openstack, and update the PR with the latest state
